hazelcast/src/main/java/com/hazelcast/internal/config/ConfigNamespace.java

```diff
@@ -66,4 +66,8 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(configSection, configName);
     }
+
+    public String getConfigSectionName() {
```

Comment:
Why do we have this? Can't you use getSectionName()? Or if you would like to, you can rename that.

Reply:
Deleted this method since we decided in the [client-protocol PR](https://github.com/hazelcast/hazelcast-client-protocol/pull/412#discussion_r794440801) that the operation will be async (hence no need to serialize/deserialize the `ConfigUpdateResult` in the client protocol).